src/main/resources/assets/js/components/AppComponent.jsx

```diff
@@ -20,6 +23,45 @@ var AppComponent = React.createClass({
       "text-warning": isDeploying
     });
 
+    var healthData = [
```

Comment:
Yes, you are right presenter logic with respect to health component. It looks much nicer in the function! Thanks